senza/traffic.py

```diff
@@ -251,7 +252,7 @@ def get_version(versions: list, version: str):
     raise click.UsageError('Stack version {} not found'.format(version))
 
 
-def get_zone(domainname: str, *args, all=False):
+def get_zone(domainname: str, *args, all=False) -> Optional[Union[Dict, List]]:
```

Comment:
`get_zone` is only used in one place now, can we get rid of it? What are the differences to Route53.get_hosted_zones?